README.md

```diff
@@ -372,6 +373,20 @@ Available `options`:
 * `ttr` - time to release in seconds. The time after which, if there is no active
 connection in the session, it will be released with all its tasks.
 
+## Session identify
+
+```lua
+queue.identify(session_uuid)
+```
+In the queue the session has a unique UUID and one session can have many
+connections. Also, the consumer can reconnect to the existing session during the
+ `ttr` time.  
+To get the UUID of the current session, call the `queue.identify()`
+without parameters.  
+To connect to the existing session, call the `queue.identify(session_uuid)`
+with the UUID of the session.  
+In case of failure, an error will be thrown.
```

Comment:
Just curious: what kinds of failures may appear here?

Reply:
For example: Trying to use invalid format UUID, expired UUID.